index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Icepace\UserHydrator;
+
+require_once "vendor/autoload.php";
```

Comment:
`require_once` should be at the top of the file before `use`

Reply:
moved